card.py

```diff
@@ -2,7 +2,28 @@ class Card:
     def __init__(self, v, s):
         self.value = v
         self.suit = s
+        self.score = self.get_score()
```

Comment:
We don't need to do this. This is what is called "denormalizing" a field. It's basically saving the result of a calculated field. If `get_score` was doing something computationally expensive we could make the argument to denormalize. But it's doing very very quick operations. So there is no need to do this.